index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Unit 2 Movie API</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="header">
+        <div id="logo_div">
+            <img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png" alt="Studio Ghilbi Logo">
+        </div>
+        <div></div>
+        <div id="header_title">
+            <h1>Ghilbi Review App</h1>
+        </div>
+    </div>
+    <div id="main_content">
+        <select onchange="loadMovieInfo(this.options[this.selectedIndex].innerText);"><option></option></select>
```

Comment:
How would you have handle the event the user generated without having to use `onchange` attribute set in the HTML. I want you to remove all `on<event-name>` on the HTML and use `addEventListener` instead. Create a commit with these changes and notify me so that I can change your grade on the FSW.4.b standard.

Reply:
select.addEventListener('change', etc...) ???

Reply:
the onchange attribute is a html built in event listener

Reply:
so is the onsubmit

Reply:
This is correct -> `select.addEventListener('change', etc...) ???`
You want to add the event listeners with JavaScript not in the HTML